package.json

```diff
@@ -1,20 +1,33 @@
 {
   "name": "react-firebase",
-  "version": "0.1.0",
+  "version": "2.0.0",
   "description": "React bindings for Firebase",
-  "main": "./lib/index.js",
+  "author": "Simen Brekken <simen@unfold.no> (http://github.com/unfold)",
```

Comment:
@einarlove Fixed in https://github.com/unfold/react-firebase/pull/6/commits/ffe5c69b066e419c7e476de8b22f8385c3fcf130

I think we should expand the author to include email and github page like in that commit